src/glint.gleam

```diff
@@ -680,6 +672,20 @@ type CommandHelp {
   )
 }
 
+/// The maximum width for console output, (e.g. command, flag, and subcommand
+/// descriptions).
+///
+const max_output_width = 100
+
+/// THe minimum width of the column containing flag/command names and the
+/// descripion.
+///
+const min_column_width = 24
+
+/// The width in spaces of a \t tab character.
+/// 
+const tab_width = 8
```

Comment:
❓ how did you arrive at this value?

Reply:
Seems to be the most common default in most terminals, however for consistency on all systems the `\t` character likely can't be used as it means the max width might not be exactly followed in all cases. Shall we remove the `\t`s?

Reply:
Yeah I think that's a good idea 👍